inspirehep/modules/records/receivers.py

```diff
@@ -63,12 +63,14 @@ def populate_inspire_document_type(recid, json):
                 elif element['primary'].lower() == 'note':
                     inspire_doc_type.append('note')
                     break
-                elif json.get('publication_info', []):
-                    for field in json.get('publication_info', []):
-                        if 'page_artid' not in field:
-                            inspire_doc_type.append('preprint')
-                            break
-
+        complete_pub_info = []
+        if not inspire_doc_type:
+            if json.get('publication_info', []):
```

Comment:
Because if publication_info doesn't exist it will return [] due to this json.get('publication_info', []) right?